backend/src/main/java/com/bakdata/conquery/commands/ManagerNode.java

```diff
@@ -183,6 +184,7 @@ public void run(ConqueryConfig config, Environment environment) throws Interrupt
 						ConqueryConfig.getInstance().getQueries().getOldQueriesTime().getUnit().toChronoUnit()
 				)));
 		environment.admin().addTask(new ClearFilterSourceSearch());
+		environment.admin().addTask(new ReportConsistencyTask(datasetRegistry));
```

Comment:
Das sollte eigentlich schon in develop drinne sein? Weißt du warum das hier auftaucht?

Reply:
ich hab mal einen rebase gemacht